cli/src/main/java/org/jboss/as/cli/impl/AttributeNamePathCompleter.java

```diff
@@ -225,7 +225,7 @@ protected AttributeNamePathCallbackHandler parse(String line) throws CommandForm
         }
 
         private boolean isWritable(ModelNode attrDescr) {
-            return !attrDescr.has(Util.ACCESS_TYPE) || !Util.READ_ONLY.equals(attrDescr.get(Util.ACCESS_TYPE).asString());
+            return !attrDescr.has(Util.ACCESS_TYPE) || !(Util.READ_ONLY.equals(attrDescr.get(Util.ACCESS_TYPE).asString()) || Util.METRIC.equals(attrDescr.get(Util.ACCESS_TYPE).asString()));
```

Comment:
I think that this condition starts to be a bit complex. It could be rewritten in a more explicit way:
private boolean isWritable(ModelNode attrDescr) {
  return !isReadOnly(attrDescr);
}
 private boolean isReadOnly(ModelNode attrDescr) {
   return attrDescr.has(Util.ACCESS_TYPE) && 
              (Util.READ_ONLY.equals(attrDescr.get(Util.ACCESS_TYPE).asString()) ||
                Util.METRIC.equals(attrDescr.get(Util.ACCESS_TYPE).asString()));
}